base/src/main/resources/db/migration/V50__folder_audit.sql

```diff
@@ -19,11 +19,98 @@
 
 -- Adds commit_audit to each folder version
 
+-- Migration function to create new dummy audits
+CREATE OR REPLACE FUNCTION ehr.migrate_folder_audit(OUT ret_id UUID) AS
+$$
+BEGIN
+    -- Add migration dummy party entry, only if not existing already
+    INSERT INTO ehr.party_identified (
+        -- id will get generated
+        name,
+        party_type,
+        object_id_type
+    )
+    SELECT 'migration_dummy',
+           'party_self',
+           'undefined'
+    WHERE NOT EXISTS (
+		SELECT 1 FROM ehr.party_identified WHERE name='migration_dummy'
```

Comment:
please replace migration_dummy by migration_dummy_65a3da3a-476f-4b1e-ab04-fa1c42edeac0

so it is unique